packages/slate-plugins/package.json

```diff
@@ -25,6 +25,7 @@
     "snyk-protect": "snyk protect"
   },
   "dependencies": {
+    "@react-hook/merged-ref": "^1.3.0",
```

Comment:
we can remove this dependency by adding the function ourselves. I don't think this paritcular function needs extra maintenance! XD

here's the function I use in my code:

```javascript
export const mergeRefs = (...refs) => {
  const filteredRefs = refs.filter(Boolean)
  if (!filteredRefs.length) return null
  if (filteredRefs.length === 0) return filteredRefs[0]
  return inst => {
    for (const ref of filteredRefs) {
      if (typeof ref === 'function') {
        ref(inst)
      } else if (ref) {
        ref.current = inst
      }
    }
  }
}
```